src/extra/audio.cpp

```diff
@@ -64,6 +64,9 @@ struct Device {
   static CBTypesInfo inputTypes() { return CoreInfo::AnyType; }
   static CBTypesInfo outputTypes() { return CoreInfo::AnyType; }
 
+  static inline TableVar _props{{"experimental", Var(true)}};
```

Comment:
Maybe the compiler optimizes but this could be shared just like params (common_types.hpp) since it's often repeated.
Same in rust.

Reply:
Btw it's just nitpick optimization

Reply:
I was thinking about that but later when we want to add other tags, how will it compose?

Reply:
I added a new constructor to allow composition.